File: python/exercício_37.py
print()
print("=-"*60)
print()
print("--------------Sistema de pulverização------------------")
print()
print("=-"*60)
print() 
print("Tipo 1 – ervas daninhas")
print("Tipo 2 – gafanhotos")
print("Tipo 3 – broca")
print("Tipo 4 – todos acima ")
n1= int(input("Digite qual desses voce deseja: "))
if n1 >= 5:
    print("Digite uma opção valida")
if n1 <=4:
    n2=int(input("Digite a quantidade de acre: "))

if n1 == 1:
    if n2 >= 1000:
        a= (50.00 * n2) * 0.95
        print("A opção escolhida foi {} com um tamanho de {}, ficando por um valor de {}".format(n1, n2, a))
        if a >= 750.00:
                aa= a - 750.00
                a = (aa * 0.90) + 750.00
                print("A opção escolhida foi {} com um tamanho de {}, ficando por um valor de {}".format(n1, n2, a))
if n1 == 2:
    if n2 >= 1000: 
        b= (100.00 * n2)* 0.95
        print("A opção escolhida foi {} com um tamanho de {}, ficando por um valor de {}".format(n1, n2, b))
        if b >= 750.00:
                bb= b - 750.00
                b = (bb * 0.90) + 750.00
                print("A opção escolhida foi {} com um tamanho de {}, ficando por um valor de {}".format(n1, n2, b))
if n1 == 3:
    if n2 >= 1000:
        c = (150.00 * n2) * 0.95
        print("A opção escolhida foi {} com um tamanho de {}, ficando por um valor de {}".format(n1, n2, c))
        if c >= 750.00:
                cc= c - 750.00
                c = (cc * 0.90) + 750.00
                print("A opção escolhida foi {} com um tamanho de {}, ficando por um valor de {}".format(n1, n2, c))
if n1 == 4:
    if n2 >= 1000:
        d = (250.00 * n2 ) * 0.95 
        print("A opção escolhida foi {} com um tamanho de {}, ficando por um valor de {}".format(n1, n2, d))
        if d >= 750.00:
                dd= d - 750.00
                d = (dd * 0.90) + 750.00
                print("A opção escolhida foi {} com um tamanho de {}, ficando por um valor de {}".format(n1, n2, d))

File: java/exercício_37.java
import java.util.Scanner;

public class exercício_37 {
    public static void main(String[] args) { 
        
        int tipo;
        double tpagar, area, sobra, valor;
        tpagar = 0; sobra = 0; valor = 0;
        
         try (Scanner leitor = new Scanner(System.in)) {
            System.out.println(" Cia de pulverizacao ");
            System.out.println("Escolha o Tipo desejado ");
            System.out.println("Tipo 1  ervas daninhas R$ 50,00 por acre ");
            System.out.println("Tipo 2  gafanhotos R$ 100,00 por acre ");
            System.out.println("Tipo 3  broca R$ 150,00 por acre ");
            System.out.println("Tipo 4  todos acima R$ 250,00 por acre ");
            
            System.out.println("Digite o numero referente a opcao desejada: ");
            tipo = leitor.nextInt();
            
            
                
                if (tipo <1 | tipo > 4){
                
                    System.out.println("OPCAO INVALIDA");
                    System.exit(0);
                }
                 
            System.out.println("Digite a quantidade de acres a serem pulverizados");
            area = leitor.nextDouble();
        }
         
        
        
            
            
            if (tipo == 1){
                
               valor = (area * 50);
            }
            
            if (tipo == 2){
            
                valor = (area * 100);
            }
            
            if (tipo == 3){
            
                valor = (area * 150);
            }

            if (tipo == 4){
            
                valor = (area * 250);
            }
            
        
            if (valor < 750.00){
            
                tpagar = valor;
            }
            
        // Se a área a ser pulverizada for superior a 1000 acres, o fazendeiro tem um desconto de 5%.
        
            if (area > 1000){
            
                valor = (valor - (valor * 0.05));
                
            } 
            
        // Se o tpgar for maior que 750,00 tem um desconto de 10% sobre o valor que ultrapassar os R$ 750,00.
        
            if (valor > 750.00){
             
                sobra = (valor - 750.00);
                
                sobra = (sobra - (sobra * 0.1));
                
                tpagar = (sobra + 750.00);
                
            }
        
        System.out.println("O valor a ser pago e de: R$ " + tpagar);
        
        
    }
}
